app/models/schedule_rule.rb

```diff
@@ -31,8 +31,8 @@ def self.unavailable_for_date(instrument, day)
     rules.each do |rule|
       res = Reservation.new({
         :product => instrument,
-        :reserve_start_at => day.dup.change(:hour => rule.start_hour, :min => rule.start_min),
-        :reserve_end_at => day.dup.change(:hour => rule.end_hour, :min => rule.end_min),
+        :reserve_start_at => day.to_time.change(:hour => rule.start_hour, :min => rule.start_min),
```

Comment:
Do you have a rubocop setup to update the hash syntax? If so, should we update this file?